src/Components/Search/Previews/Grids/ArtistSearch/NoResults.tsx

```diff
@@ -0,0 +1,42 @@
+import { Flex, Serif } from "@artsy/palette"
+import React from "react"
+
+const ArtworkIcon: React.SFC = () => {
+  return (
+    <svg width="18px" height="18px" viewBox="2 2 14 14" version="1.1">
```

Comment:
I'm going to follow up with @jeffreytr about the viewBox on this icon, and possibly others. It had some whitespace around the edges.